tests/daemon/test-daemon.c

```diff
@@ -1225,48 +1175,13 @@ test_daemon_does_not_record_sequences_when_daemon_disabled (Fixture      *fixtur
 }
 
 static void
-test_daemon_updates_timestamps_on_shutdown (Fixture      *fixture,
```

Comment:
What I'm trying to say is that we should add a test that does nothing more and nothing less than verify that calling `g_clear_object (&fixture->test_object)` causes `fixture->mock_persistent_cache` to be unreffed. The underlying reason for this test is that leaking the persistent cache would have more significant consequences than the standard waste of memory; it's actually a timestamp data quality concern. Does that make sense?